opensearch-operator/pkg/helpers/helpers.go

```diff
@@ -359,3 +360,28 @@ func CompareVersions(v1 string, v2 string) bool {
 	ver2, _ := version.NewVersion(v2)
 	return err == nil && ver1.LessThan(ver2)
 }
+
+func GetJvmHeapSize(nodePool *opsterv1.NodePool) string {
+	jvmHeapSizeTemplate := "-Xmx%s -Xms%s"
+
+	if nodePool.Jvm == "" {
+		memoryLimit := nodePool.Resources.Requests.Memory()
+		nodePoolMemorySize, _ := resource.ParseQuantity(memoryLimit.String())
```

Comment:
`memoryLimit` is already a quantity, why convert to String and parse again?

Reply:
I didn't know that. I refactored the function to exploit that fact and simplify the logic.